posts/inside-rust/2023-12-08-cargo-cache-cleaning.md

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
I'd personally be in favor of putting this on the main blog, not inside Rust, similar to the parallel-rustc nightly announcement.

Reply:
Sure! I have moved it over.